gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-5.5.1-all.zip
```

Comment:
This is auto-generated, so `:` must probably be kept escaped.

Reply:
But it doesn’t work when I escape :  
Then, is it better not to fix this one ?-?

Reply:
> But it doesn’t work when I escape

What's your error message?

Reply:
Actually it’s not error, it’ s not connected right away.

Reply:
What is "not connected"?

Reply:
Umm I mean, when I type https\://services.gradle.org/distributions/gradle-5.5.1-all.zip , it is just searched on the web. But when I type https://services.gradle.org/distributions/gradle-5.5.1-all.zip , it is downloaded directly.

Reply:
This file is parsed and interpreted by gradle, some characters are escaped probably to work with same separators accepted by the format, but in the end they are unescaped to retrieve the content.

Reply:
Then, is it better to useoriginal one?

Reply:
Yes.